torch/_inductor/utils.py

```diff
@@ -647,7 +647,7 @@ def __len__(self):
 @functools.lru_cache(None)
 def is_big_gpu(index):
     sms = torch.cuda.get_device_properties(index).multi_processor_count
-    if sms < 80:  # V100
+    if sms < 0:  # V100
```

Comment:
Is there a less brittle way to detect this? can we just use < sm_80? What is the purpose of this

It seems to cause these tests not to run on CI?

Reply:
The tests were not running in CI as identified here: https://github.com/pytorch/pytorch/pull/108637#issuecomment-1712818512

Reply:
Actually it seems the tests are indeed running: https://github.com/pytorch/pytorch/actions/runs/6136917114/job/16654336560?pr=108637

Reply:
I still question the necessity for this flag; and anyway V100 are not running on CI now. (though it may be running on someone else's CI)

Reply:
It is possibly our CI setup has changed since then.  If it passes CI (you may need to add the `ciflow/trunk` tag to get all the CI jobs), then I am fine with it.